dist/t/spec/features/0040_package_spec.rb

```diff
@@ -26,10 +26,8 @@
       click_link('Your Home Project')
     end
     click_link('hello_world')
-    click_link('Add File')
-    attach_file('file', File.expand_path('../fixtures/hello_world.spec', __dir__), make_visible: true)
-    click_button('Add File')
-    expect(page).to have_content("The file 'hello_world.spec' has been successfully saved.")
+    attach_file('files', File.expand_path('../fixtures/hello_world.spec', __dir__), make_visible: true)
```

Comment:
```suggestion
    attach_file('files[]', File.expand_path('../fixtures/hello_world.spec', __dir__), make_visible: true)
```
Does it also work without the brackets?